editor/src/clj/internal/node.clj

```diff
@@ -1507,7 +1507,8 @@
 (defn- schema-check-result-form [description label node-id-sym label-sym evaluation-context-sym result-sym forms]
   (if *check-schemas*
     `(do
-       (schema-check-result ~node-id-sym ~label-sym ~evaluation-context-sym ~(deduce-output-type-form description label) ~result-sym)
+       (when ~`*check-schemas* ; Inner check to support disabling the schema check post compile-time.
+         (schema-check-result ~node-id-sym ~label-sym ~evaluation-context-sym ~(deduce-output-type-form description label) ~result-sym))
```

Comment:
Hmm, this change worries me, is it necessary? This code will be injected into every output, increasing the code size and potentially affecting jit. I also don't see `*check-schemas*` being rebound anywhere...

Reply:
I'm merely wrapping the existing call in `(when *check-schemas* ...)`. Do you think it will have much of an impact? This is called at compile-time to generate the forms, so the additional `(when *check-schemas* ...)` check only applies to `:dev` builds that have `*check-schemas*` enabled. If `*check-schemas*` is `false` at compile time, which it is in `:release` builds, the `schema-check-result-form` function returns no forms at all.

`*check-schemas*` is being rebound here in `perf_test.clj`: https://github.com/defold/defold/blob/editor-dev/editor/test/integration/perf_test.clj#L47